frontend/packages/portal-frontend/src/dAPI.ts

```diff
@@ -1096,6 +1096,22 @@ export class DepmapApi {
     return Promise.reject(Error("Wrong api used. Check ApiContext"));
   }
 
+  getTabularDatasetData(
+    datasetId: string,
+    args: {
+      identifier: "id" | "label";
+      columns?: string[] | null;
+    }
+  ): Promise<{ [key: string]: { [key: string]: any } }> {
+    return Promise.reject(Error("Wrong api used. Check ApiContext"));
```

Comment:
I have a branch where I've removed `ApiContext` and completely revamped the API. We won't need to do any of this "wrong api!" nonsense anymore. I'm going to wait to merge it until all your stuff is done. Otherwise you'd have some horrible merge conflicts to resolve.